src/lualib/Error.ts

```diff
@@ -0,0 +1,78 @@
+type TSTLCapturedErrorStack = Array<{
```

Comment:
It's not really tstl-specific, it's just a result of debug.getinfo, so it should be added to debug declarations (see https://github.com/ark120202/lua-types/blob/master/core/debug.d.ts#L39).

Reply:
This case it is specific though as it represents the exact parameters requested from `debug.getinfo(level, "Snl");`.